synapse/res/templates/sso_redirect_confirm.html

```diff
@@ -3,12 +3,34 @@
 <head>
     <meta charset="UTF-8">
     <title>SSO redirect confirmation</title>
+    <meta name="viewport" content="width=device-width, user-scalable=no">
+    <style type="text/css">
+      {% include "sso.css" without context %}
+    </style>
 </head>
     <body>
-        <p>The application at <span style="font-weight:bold">{{ display_url | e }}</span> is requesting full access to your <span style="font-weight:bold">{{ server_name }}</span> Matrix account.</p>
-        <p>If you don't recognise this address, you should ignore this and close this tab.</p>
-        <p>
-            <a href="{{ redirect_url | e }}">I trust this address</a>
-        </p>
+        <header>
+            <!-- {% if new_user %} -->
```

Comment:
Why putting this in HTML comment tags?

Reply:
to make it it easier to develop outside the template engine, I believe.

Reply:
I've never seen this pattern before. What makes it easier to develop without these? We don't seem to do this in any other templates. Personally I find it makes my eyes ignore the Jinja logic.